README.md

```diff
@@ -1 +1,21 @@
-# git-practice-pizza
\ No newline at end of file
+# git-practice-pizza
+
+## Pasos para crear una PIZZA
+
+###### Ingredientes
+
+Harina de trigo
+Agua
+Mantequilla
+Sal y azúcar al gusto
+Salsa de tomate
+Piña en trozos
+Jamón
+Levadura
+
+###### Preparación
+
+1. Cernir el harina y hacer una fuente, agregar la levadura, la sal, el azúcar, y revolver. Formar nuevamente la fuente y agregar la mantequilla, agua tibia y empezar a amasar hasta que la masa tenga una textura suave (ampollitas). Colocar la masa en un recipiente y taparla con una bolsa para dejarla reposar hasta que doble el volumen (dejarla en un lugar caliente o al sol).
+
+2. Agregar todos los ingredientes de la salsa al puré de tomate y sazonar.
```

Comment:
If any other steps are added is better to use `1.` so new steps will be numbered automatically